### PR TITLE
[#86] Update the "try lite now" SVG badge and add a "launch lite" SVG badge

### PR DESCRIPTION
**Pull request by xiaohk**
## References

This PR makes changes based on [the comment](https://github.com/jupyterlite/jupyterlite/issues/86#issuecomment-1060060371) in #86.

## Code changes

|Previous Badge|New Badge (minor change)|Alternative Badge|
|:---:|:---:|:---:|
|badge|badge|badge-launch|

1. Replace `/docs/_static/badge.svg` to add more horizontal space, minor color change (text shadow, "jl" in the lightbulb), and smaller SVG file size (24kb to 7kb)
2. Add `/docs/_static/badge-launch.svg` as an alternative badge for users to link to their JupyterLite demo page (similar to "launch binder")

## User-facing changes

Pages that have references to `/docs/_static/badge.svg`  (documentation, README, etc) will display the new badge SVG.

## Backwards-incompatible changes

N/A

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks!

The "launch lite" badge fits pretty well next to the "launch binder" badge.

cc @bollwyvl who might want to have a look too

File: docs/_static/badge.svg [hunk 1]
```diff
@@ -1,315 +1 @@
...
...13 319.629,316.547 319.629,316.995C319.629,317.182 319.581,317.365 319.485,317.543C319.269,317.939 318.936,318.137 318.488,318.137C318.299,318.137 318.116,318.089 317.94,317.993C317.534,317.787 317.331,317.454 317.331,316.995Z" style="fill:rgb(85,85,85);fill-rule:nonzero;"/></g></g></g><g transform="matrix(2.92212,0,0,14.065,10424.3,-12659.1)"><g transform="matrix(0.295204,0,0,0.0956347,2538.31,1594.22)"><text x="-181.194px" y="0px" style="font-family:'Verdana', sans-serif;font-size:130.5px;fill:rgb(1,1,1);fill-opacity:0.3;">try</text></g><g transform="matrix(0.295204,0,0,0.0956347,2538.31,1593.26)"><text x="-181.194px" y="0px" style="font-family:'Verdana', sans-serif;font-size:130.5px;fill:white;">try</text></g></g><g transform="matrix(3.46032,0,0,16.6556,9337.23,-16787)"><g transform="matrix(0.295204,0,0,0.0956347,2538.31,1594.22)"><text x="-181.194px" y="0px" style="font-family:'Verdana', sans-serif;font-size:110px;fill:rgb(1,1,1);fill-opacity:0.1;">lite now</text></g><g transform="matrix(0.295204,0,0,0.0956347,2538.31,1593.26)"><text x="-181.194px" y="0px" style="font-family:'Verdana', sans-serif;font-size:110px;fill:rgb(56,56,56);">lite now</text></g></g></g></g></g></g></svg>
```
Comment (jtpio) [inline docs/_static/badge.svg:1]: Maybe we'll want to keep the non-minimized svg here? (or keep it separate)

**Review by bollwyvl — APPROVED** [review]

**Comment by jtpio** [comment]
Thanks!

**Comment by xiaohk** [comment]
Thank you @bollwyvl, @jtpio! 🙏

**Pull request merged**